### DISCUSSION TITLE
Is Masque Supposed to Skin the Blizzard Player Buff/Debuffs?

### DISCUSSION THREAD

**Discussion by Chzconey** [SFX-WoW/Masque#125] [Questions] [answered]
Was wondering if it is supposed to skin the player buff bar or not.
![Screenshot_1](https://user-images.githubusercontent.com/24687655/98982216-01254180-24ed-11eb-8f9a-57b958e61ee3.png)

**Comment by StormFX** [comment, marked as answer]
No. Masque doesn't skin any default frames. You'll need a buff frame add-on that supports Masque.

**Comment by Chzconey** [comment]
Thanks for answering. Are there any in particular that work with Masque?

**Comment by StormFX** [comment]
There's one called "Masque Skinner: Blizz Buffs" or something.

**Comment by Chzconey** [comment]
Thanks again, it just wasn't updated didn't know if it would work.

**Comment by StormFX** [comment]
Last I checked it works.
